src/main/java/com/google/sps/data/Member.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+
+//Member object representing members of the Room
+
```

Comment:
nit: remove empty line. No line needed btwn javadoc and the java class.

Reply:
Not don.e